version.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package firecracker
+
+// Version represents the current version of the SDK.
+const Version = "v0.15.0"
```

Comment:
I think we don't need `v` here. [Semver](https://semver.org/) expects `MAJOR.MINOR.PATCH` format.

Reply:
Yep! Good catch